static/app/views/replays/detail/eventMetaData.tsx

```diff
@@ -19,16 +19,19 @@ type Props = {
 };
 
 function EventMetaData({crumbs, durationMs, replayRecord}: Props) {
-  const {projects} = useProjects();
+  const {
+    params: {replaySlug},
+  } = useRouteContext();
   const errors = crumbs?.filter(crumb => crumb.type === 'error').length;
 
   return (
     <KeyMetrics>
       {replayRecord ? (
         <ProjectBadge
           project={
-            projects.find(p => p.id === replayRecord.projectId) || {
-              slug: replayRecord.projectSlug || '',
+            replayRecord.project || {
```

Comment:
Yeah I think we'll want to use `useProjects` here instead and use the loading state from the hook.

Reply:
fair enough. also it looks like this is the only spot that reads `.project`